vocab/credentials/v2/vocabulary.yml

```diff
@@ -45,56 +50,56 @@ class:
     label: Credential evidence
     comment: A Credential Evidence scheme provides enough information to a verifier to determine whether the evidence gathered meets their requirements for issuing a credential. The precise content of each evidence scheme is determined by the specific evidence type definition.
 
+  - id: CredentialGraph
+    label: (Verifiable) credential graph
+    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one <a href="#Credential">Credential</a> (usually a <a href="#VerifiableCredential">Verifiable Credential</a>).
+
   - id: VerifiableCredential
     label: Verifiable credential
     comment: |
-      A Credential is a set of one or more claims made by an issuer. A Verifiable Credential is a tamper-evident credential that has authorship that can be cryptographically verified. Verifiable Credentials can be used to build Verifiable Presentations, which can also be cryptographically verified.
-
-  - id: VerifiableCredentialGraph
-    label: Verifiable credential graph
-    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one Verifiable Credential
+      A Verifiable Credential is a <a href="#Credential">Credential</a> that is tamper-evident and has authorship that can be cryptographically verified. Verifiable Credentials can be used to build Verifiable Presentations, which can also be cryptographically verified.
```

Comment:
This seems like an improvement... but I note the conflict with https://github.com/w3c/vc-data-model/pull/1057/files#r1124649817

Reply:
Conflict?

Reply:
Sorry, more detail would have helped here...

<img width="1240" alt="Screen Shot 2023-03-10 at 4 43 10 PM" src="https://user-images.githubusercontent.com/8295856/224441925-df8aac8f-7ab8-47da-b80c-f1455db7c462.png">

Are VerifiablePresentations made of Credentials?

Are VerifiablePresentations made of VerifiableCredentials?

Are Presentations made of VerifiableCredentials?

Are Presentations made of Credentials?

If a VerifiableCredential is a Credential, and a VerifiablePresentations is a Presentation....

These are the conflicts I mean....

The current spec defines "VerifiablePresentations" in terms of "VerifiableCredentials", you noted the confusion here: 

- https://github.com/w3c/vc-data-model/pull/1057#discussion_r1126533110

Reply:
Whenever possible, when presenting text, please present it *as* text. It is difficult to read the screencap, and impossible to copy-and-paste from it, which is likely to be helpful in suggesting revisions (on either side of the flagged conflict).